recipes/bacnet-stack/all/conandata.yml

```diff
@@ -1,10 +1,13 @@
 sources:
-  "20200306":
+  "cci.20200306":
     sha256: bf61bc7ffe556b30464ac0734e18d975fe6a8676d9c2fcb7c215981a46381847
     url: https://github.com/bacnet-stack/bacnet-stack/archive/4a916468c63de478b84ef4d0c67d541cd84da27e.zip
-  "20200515":
+  "cci.20200515":
     sha256: dda1b8c180c3eeb85eb950de5a5907fa22543d7c6b930f11f3fe735eb6ea662b
     url: https://github.com/bacnet-stack/bacnet-stack/archive/3553ae56c2b0240949ddf54dbea4213b0c69433c.zip
-  "20200601":
+  "cci.20200601":
```

Comment:
Oh no. Not yet another open question for versioning schemes 😓  

I don't think that you should rename the versions. Since we don't de-publish existing versions from the Conan repository, this only removes these version from the  latest git checkout.

The damage, that we are mixing dates and upstream version numbers, is pretty much already done (assuming that we are now adding the upstream `1.0.0` version, but I don't see why we should not)

Reply:
I dont see any problem: we keep the okd versions e.g. „20200601“ and issue a warnig for it.
We also add „cci.20200601“ which is the usage on new conan recipes
And add semver foe upstream releases

Reply:
Keep the old names, too late for moving it, you can break users which are using that version.